content/articles/setting-up-accounts-for-clients.markdown

```diff
@@ -15,32 +15,32 @@ categories:
 ---
 
 <info>
-This article describes a feature that is only available to the following [new plans](/articles/new-plans#newer-plans-some): Professional and Business.
+This article describes a feature only available on the following [plans](/articles/new-plans#newer-plans-some): Professional and Business.
```

Comment:
@itsalyse this notice is actually reused in multiple places across this site. Should we adjust all those instances to keep them consistent?

Reply:
Yeah. I've been doing that as I go through all the support articles.